docs/reference/aggregations/bucket/diversified-sampler-aggregation.asciidoc

```diff
@@ -83,10 +84,10 @@ Response:
 }
 --------------------------------------------------
 // TESTRESPONSE[s/\.\.\./"took": $body.took,"timed_out": false,"_shards": $body._shards,"hits": $body.hits,/]
-// TESTRESPONSE[s/1000/151/]
 // TESTRESPONSE[s/2.213/$body.aggregations.my_unbiased_sample.keywords.buckets.0.score/]
 
-<1> 1000 documents were sampled in total because we asked for a maximum of 200 from an index with 5 shards. The cost of performing the nested significant_terms aggregation was therefore limited rather than unbounded.
+<1> 151 documents were sampled in total because we asked for a maximum of 200 from an index with 5 shards. The cost of performing the
+nested significant_terms aggregation was therefore limited rather than unbounded.
```

Comment:
I don't understand this explanation anymore, how is 151 related to a maximum on 200 documetns and 5 shards?